src/main/java/org/opendevstack/provision/services/JenkinsPipelineAdapter.java

```diff
@@ -65,8 +65,11 @@
 
   public static final String OPTION_KEY_GIT_SERVER_URL = "GIT_SERVER_URL";
 
-  @Value("${openshift.jenkins.webhookproxy.name.pattern}")
-  protected String projectOpenshiftJenkinsWebhookProxyNamePattern;
+  @Value("${openshift.jenkins.admin.webhookproxy.host}")
+  protected String adminWebhookProxyHost;
```

Comment:
any reason why not use `private` instead?

Reply:
It's aligned with the visibility of the other attributes in this class.

Reply:
ok, then can you please change it to `private`?

Reply:
It's not possible bcs. of how the Test is written. See lines starting from https://github.com/opendevstack/ods-provisioning-app/blob/7db613174625a454dc1a7ed9b9ca605b5886e9a5/src/test/java/org/opendevstack/provision/services/JenkinsPipelineAdapterTest.java#L73

Reply:
ok, actually this is an anitpattern in java... the preferred way is to use getters and setters in such cases. We can leave it this way for now.